specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/storage.json

```diff
@@ -1628,6 +1628,25 @@
           "x-ms-client-name": "FailoverInProgress",
           "description": "If the failover is in progress, the value will be true, otherwise, it will be null.",
           "readOnly": true
+        },
+        "minimumTLSVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "TLS version control on storage account in customer configuration, TLS v1.2 by default.",
+          "enum": [
+            "TLS1_0",
+            "TLS1_1",
+            "TLS1_2"
+          ],
+          "x-ms-enum": {
+            "name": "MinimumTLSVersion",
+            "modelAsString": false
```

Comment:
ModelAsString in most cases. Otherwise the property becomes an enum type in generated SDK. Then when we introduce a new enum value in newer versions it breaks older client because the SDK would fail deserializing the new value.